src/components/DeploymentQuickRunOverflowMenuItem.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <p-overflow-menu-item @click="run">
+    <slot>
+      Quick run
+    </slot>
+  </p-overflow-menu-item>
+</template>
+
+<script lang="ts" setup>
+  import { showToast } from '@prefecthq/prefect-design'
+  import { h } from 'vue'
+  import { useRouter } from 'vue-router'
+  import ToastFlowRunCreate from './ToastFlowRunCreate.vue'
+  import { useWorkspaceApi, useWorkspaceRoutes } from '@/compositions'
+  import { localization } from '@/localization'
+  import { Deployment } from '@/models'
+
+  const props = defineProps<{
+    deployment: Deployment,
```

Comment:
You're only using the id in this component so I would accept just the `deploymentId` similar to the other menu item.